src/classes/pilot/Pilot.ts

```diff
@@ -1221,15 +1221,16 @@ class Pilot implements ICloudSyncable {
     }
   }
 
-  public Update(data: IPilotData, ignoreProps?: boolean): void {
-    if (ignoreProps) {
+  public Update(data: IPilotData, sync?: boolean): void {
+    if (sync) {
```

Comment:
Turn the old "ignoreProps" parameter into a new "sync" parameter. This was only set when performing a could sync right now anyway.